src/Generators/Microsoft.Gen.Logging/Emission/Emitter.Method.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.Gen.Logging.Emission;
 
 internal sealed partial class Emitter : EmitterBase
 {
+    private const string PropertySeparator = ".";
+    private const string NullablePropertySeparator = "?.";
```

Comment:
I'm not thrilled with this NUllablePropertySeparator. PropertySeparator is used when we're producing a name string to be used in telemetry, so the string could in fact be anything we want and not break anything. However, NullablePropertySeparator is used to produce source code that references a particular member in the generated code. So that can't readily be changed as it would break the generated code.

Maybe just add some comments to deacribe what these are for...

Reply:
I didn't know that the string constants were inlined intentionally, therefore when making changes I extracted them into constant fields following "best practices". I removed the constant fields and inlined the constant strings.